definitions/tools/optitype_dna.wdl

```diff
@@ -7,19 +7,22 @@ task optitypeDna {
     File cram_crai
     File reference
     File reference_fai
+    Int threads = 8
+    Int mem = 50
   }
 
   Int space_needed_gb = 10 + round(5*size([cram, cram_crai, reference, reference_fai], "GB"))
   runtime {
-    memory: "64GB"
-    docker: "mgibio/immuno_tools-cwl:1.0.1"
+    memory: "~{mem}GB"
+    cpu: threads 
+    docker: "laljorani20/immuno-tools:latest"
     disks: "local-disk ~{space_needed_gb} HDD"
     bootDiskSizeGb: 3*space_needed_gb
   }
 
   command <<<
-    /bin/bash /usr/bin/optitype_script.sh /tmp . \
-    ~{optitype_name} ~{cram} ~{reference}
+    /bin/bash /usr/bin/optitype_script_wdl_improved.sh /tmp . \
```

Comment:
Can we replace the existing script instead of having a second one alongside it?  Could make it backwards-compatible by setting default mem and thread numbers in the shell script or maybe by revising the CWL version of the tool, too.

If not it might be nice if the name is descriptive of when to use it versus the original.  (Of course, the original name `optitype_script.sh` was already a bit redundant, too 🙂)  I see there's a third script in the repo named `optitype_script_wdl.sh`--that's exactly the sort of proliferation of versions I'd be hoping to avoid by asking this.